llm_groq.py

```diff
@@ -9,13 +9,15 @@
 def register_models(register):
     register(LLMGroq("groq-llama2"))
     register(LLMGroq("groq-mixtral"))
+    register(LLMGroq("groq-gemma"))
```

Comment:
does it make sense to put these in alphabetical order?

Reply:
That's a good question. Sorry, for just getting around to your PR now :(